app/src/main/java/com/pajato/android/gamechat/main/PaneManager.java

```diff
@@ -44,7 +45,8 @@
     private static final String TAG = PaneManager.class.getSimpleName();
 
     /** The fragment list index for the game fragment. */
-    private static final int GAME_INDEX = 1;
+    public static final int CHAT_INDEX = 0;
```

Comment:
I am gradually evolving a standard ordering to Java constructs.  First comes enums, then class constants, class variables, instance constants (hardly ever see any of these), instance variables, constructors, instance methods and inner classes.  Within each section, I tend to order them public, protected and then private.  Fwiw.